packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -91,6 +91,7 @@ export default function useTabNav() {
 	const ref = useRefEffect( ( node ) => {
 		function onKeyDown( event ) {
 			if ( event.keyCode === ESCAPE && ! hasMultiSelection() ) {
+				event.stopPropagation();
```

Comment:
How do we handle this in the post editor? We should really avoid `stopPropagation`.

Reply:
I think it has a valid use case for Escape key, but I see your point!

> How do we handle this in the post editor?

I think this hook is used in the post editor?

Reply:
Right, it is used in the post editor, so I wonder how it worked fine there before?

Reply:
Ah, I see now:

> It should enter navigation mode without closing the editor.

I usually use `event.preventDefault()` and then check down the line if it's been default prevented already. Anyway this seems fine since we're already doing it in other places.